libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -29,7 +29,7 @@
 // The maximum we need to store is gyro-rate / loop-rate, worst case ArduCopter with BMI088 is 2000/400
 #define INS_MAX_GYRO_WINDOW_SAMPLES 8
 
-#define DEFAULT_IMU_LOG_BAT_MASK 0
+#define DEFAULT_IMU_LOG_BAT_MASK 3
```

Comment:
Don't this enable batch logging everywhere?

Reply:
Fixed, thanks